assets/src/app/colors.less

```diff
@@ -13,4 +13,4 @@
 @lightGrey: #eee;
 @purple: #671368;
 @yellow: #ff0;
-@orange: #ffa500;
+@orange: #f96300;
```

Comment:
don't change that, add a new color name deepOrange or something like that.

Reply:
I replaced this color because it is never used in the code .